include/display/README.md

```
@@ -0,0 +1,71 @@
+# Littlev Graphics Libraray
+
+![LittlevGL cover](http://www.gl.littlev.hu/home/main_cover_small.png)
+
+LittlevGL provides everything you need to create a Graphical User Interface (GUI) on embedded systems with easy-to-use graphical elements, beautiful visual effects and low memory footprint.
+
+Homepage: https://littlevgl.com
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)